content/_data/changelogs/weekly.yml

```diff
@@ -93,6 +93,10 @@
       pull: 2683
     - type: rfe
       message: >
+        Searching in the <code>Build History</code> widget takes into account user preferences (case sensitivity by default).
```

Comment:
For me it is important to let people know why search stopped working unexpectedly in some cases (e.g. failed vs FAILED). https://github.com/jenkinsci/jenkins/pull/2801 is only for new and anonymous users. For all existing users for radical changes are required - as proposed in [JENKINS-42701](https://issues.jenkins-ci.org/browse/JENKINS-42701) for which I don't have a good idea.